native/src/modules/endpoint/hocs/withPayloadProvider.js

```diff
@@ -94,6 +94,7 @@ const withPayloadProvider = <S: { dispatch: Dispatch<StoreActionType> }, R: {}>
           return <LanguageNotAvailableContainer languages={props.availableLanguages}
                                                 changeLanguage={this.changeUnavailableLanguage} />
         } else if (props.status === 'loading') {
+          // TODO: console.log('render loading', props.progress)
```

Comment:
Also here? [IGAPP-292](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=14&view=detail&selectedIssue=IGAPP-292): Notify user if offline when refreshing

Reply:
Yes, true, I just saw that this is the place for implementingIGAPP-292. But it's a completely different Task and does not belong here...